Feature: Work - About

luftedar opened the pull request:
### My Portfolio Project Header and Main

In this update:

-  Updated HTML Semantic Tags for Work and About Section.
-  Updated CSS for Work and About Sections. Grids and Flexboxes are Used.
-  Updated README.md

barackm commented on index.html:
```diff
@@ -41,6 +41,128 @@ <h1>Hey there. I’m Orçun<br>
+                    <h2 class="section-header">My Recent <br> Works</h2>
+                    <div id="green-line"></div>
+                </div>
+                <div id="projects-cards">
```
- [x] Nice job using `<section>` tags for the sections of your webpage, However, there is a couple of things we will need to fix 😉 . According to the project requirement, we are supposed to have two `<section>` tags, one for the ***Works*** section and the other for the **About myself** section, and they should be independent, but for now, I can see that you have multiple `<section>` tags and the **About myself** section is also inside the **Works** section, Kindly let's only have for now two `<section>` tags for the Works and About myself, and remember the about myself section should not be inside the Works section. 👍 


- [x] Good idea using the `display: grid`, but you did not use it at the right place, remember we need to use the `display: grid` to align the projects, see screenshot below 👇 
![image](https://user-images.githubusercontent.com/55465133/133660577-0bc1a8b9-858d-4fa0-ad49-7ff1c219ea69.png), Kindly let's have the `display: grid` on the container of the projects 😉

barackm commented on index.html:
```diff
@@ -41,6 +41,128 @@ <h1>Hey there. I’m Orçun<br>
+                                    <li class="card-list-element">HTML</li>
+                                </ul>
+                            </div>
+                            <div><button class="card-button">See Project</button></div>
```
- [x] Let us please have button interactions like Active, Hover, and Pressed on all the buttons, see the design below 👇 
![image](https://user-images.githubusercontent.com/55465133/133661305-453aeea4-5035-44eb-848a-f19288346800.png)

luftedar replied:
Hello. I want to be sure about this one. Is it enough to use :hover(for hover) and :active(for pressed)? Is there anything else for enabled? Should I also use the :visited?

barackm replied:
Hi @luftedar, you can add other interactions if you wish, but the ones which are mandatory are Active, Hover, and Presse. and remember enabled is the default one so you don't need to specify it. Yes, you can use `:active` for pressed 😉

luftedar replied:
Thanks @barackm you helped me a lot.  :100:

barackm replied:
You're welcome @luftedar, Keep up the good work 😉

barackm commented on style.css:
```diff
@@ -11,26 +15,43 @@ header {
 }
 
 hgroup {
   display: flex;
```
- [x] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

barackm reviewed (changes requested):
Hi @luftedar, 👋 

You have done an amazing job so far! 👏👏👏
However, there are some issues that you still need to work on to go to the next project. But remember you are almost there 😉  😉 !

## To Highlight
✔️  Nice PR title
✔️  Nice PR summary
✔️  Good use of GitHub flow 👏 

# STATUS: Required Changes ♻️
### _Check the comments under the review._ 👇 
![](https://media.giphy.com/media/iZKF7CKpEEva2OZE6a/giphy.gif)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

luftedar commented:
###My Portfolio Project Header and Main:

In this update for requirements:

- Fixed Structure of Semantic Tags.
- Updated button hover and pressed effects.
- Updated cards' parent element to grid.
- Updated cards elements to flex-box.
- Applied DRY coding rules.

TedLivist commented on style.css:
```diff
@@ -1,39 +1,80 @@
+}
+
+nav {
+  display: flex;
```
- [x] In following with the DRY principle introduced by the previous reviewer, you did well by extracting the `display: flex` and `flex-direction: column` properties. But you seem to be using it anyways in these stylings. This defeats the purpose of the extraction. Please remove those properties in the elements that have them defined in and add them as class attributes in the HTML tags where appropriate.

For example, header should be
```
header {
  background: #fff;
}
```
and HTML should be
```
<header class="d-flex d-flex-column">
```

TedLivist commented on index.html:
```diff
@@ -42,6 +41,132 @@ <h1>Hey there. I’m Orçun<br>
+                        </div>
+                    </div>
+                    
+
```
- [x] As defined in the [HTML & CSS best practices link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):

![blank_m](https://user-images.githubusercontent.com/36542986/133823821-77f347a4-b184-4770-998f-9b0779c54b6d.jpg)

Please remove this spacing. Removing it makes your code more readable.

TedLivist commented on README.md:
```diff
@@ -15,7 +15,7 @@ In this project, semantic HTML5 tags used while positioning for CSS made by usin
 ## Live Demo
 
-[Live Demo Link](https://raw.githack.com/luftedar/My-Portoflio/feature/index.html)
+[Live Demo Link](https://raw.githack.com/luftedar/My-Portoflio/feature-work-about/index.html)
```
- [ ] There seems to be something wrong with the display of your website and live link which was not present during the previous review. Please fix it. I have added more descriptions below.

Your project before the first review was just fine and scrollable. I can scroll down to the part of the page you can see in the image:

![old_scr](https://user-images.githubusercontent.com/36542986/133826086-c4173a63-a8a0-407d-83b7-6450f0889543.jpg)

But your project now is not scrollable at all. I have checked it on mobile and desktop. I can only see this part of the page by default: 

![new_scr](https://user-images.githubusercontent.com/36542986/133826425-4812a1c8-9598-408d-a849-ad87e51fa1ba.jpg)

and I need to zoom out before I can start seeing the `Recent Works` title. Even at that, I still cannot scroll to the bottom of the page like in the previous image.

luftedar replied:
Thanks for your review. My last push is not shown here but it was successful in my terminal. :disappointed: . I need to practice more for the git.

TedLivist replied:
Alright. Sure, over time you'll get the hang of it.

TedLivist reviewed (changes requested):
Hi @luftedar,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

luftedar commented:
###My Portfolio Project Header and Main:

In this update for requirements:

 -   Fixed Structure of Semantic Tags.
 -   Updated button hover and pressed effects.
 -   Updated cards' parent element to grid.
 -   Updated cards elements to flex-box.
 -   Applied DRY coding rules.

rukundoeric reviewed (approved):
Hi @luftedar,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
